elastic/integrations
# [Beaconing] Packaging Network Beaconing Detection

## What does this PR do?
Package the beaconing detection model previously installed via dreml repo.


## Checklist


## Author's Checklist


Points to review:
- The README covers the description and steps to configure this package
- Check for `transform.yml` and dashboard objects
- Manifest has the right license version and description. Changelog points to correct PR. 

## How to test this PR locally


Use `elastic-package build` to build the package and stand up the stack locally using the following command:

                                        elastic-package stack up -d -v --version 8.8.0

Make sure the transform is installed and started before checking the dashboards.

## Related issues



## Screenshots


Screenshots are tracked in the comments below.
